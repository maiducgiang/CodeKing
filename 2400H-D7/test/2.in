6
30 5
1 3
20 7
31 20
25 10
49 30
