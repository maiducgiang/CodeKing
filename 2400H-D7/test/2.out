144168843
1
799471640
289924606
877446114
87227546
